src/devices/machine/netlist.h

```diff
@@ -57,6 +57,47 @@ namespace netlist {
 	void _name(const int data, const attotime &time)
 
 
+// ----------------------------------------------------------------------------------------
+// netlist_log_csv
+// ----------------------------------------------------------------------------------------
+
+template <int USE>
+struct netlist_log_csv
+{
+private:
+	static constexpr int MAX_BUFFER_ENTRIES = 1000;
+public:
+	void open(running_machine &machine, const std::string &name) const { }
+	void close() const { }
+	void log_add(char const* param, double value, bool isfloat) const { }
+	void log_flush(int count = MAX_BUFFER_ENTRIES) const { }
```

Comment:
This doesn’t provide the same interface as the enabled version – the member functions are not `const` qualified in the enabled version.  This makes development more error-prone, as someone with the feature disabled may inadvertently add a call to a member function on a `const` instance of a logger.  The enabled and disabled versions should provide identical interfaces.

Reply:
I have fixed this now.